packages/@aws-cdk/aws-autoscaling/test/integ.asg-metrics-collections.ts

```diff
@@ -0,0 +1,29 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as cdk from '@aws-cdk/core';
+import * as autoscaling from '../lib';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-cdk-asg-integ');
+
```

Comment:
Curious: Is there value in having an integration test for this change? Feels like unit tests should be sufficient